__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -468,6 +498,7 @@ describe('TESTING ChatSchema', () => {
         isMe: true,
         message: 'message',
         rawJson: null,
+        attachmentJson: '[]',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachmentJson` field is initialized as a string representation of an empty array. This is correct if we are storing JSON strings in the database.



<!-- This is an auto-generated comment by OSS CodeRabbit -->